setup.py

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env python
 
-from git_machete import __version__
 from os import path
 from setuptools import setup
 
 this_directory = path.abspath(path.dirname(__file__))
 with open(path.join(this_directory, 'README.md')) as f:
     long_description = f.read()
 
+with open(path.join(this_directory, 'VERSION')) as f:
```

Comment:
Does the consideration from https://github.com/crdoconnor/strictyaml/blob/master/setup.py#L12 apply here as well? 🤔 (same for reading README.md above)